codec/src/main/java/io/netty/handler/codec/compression/Brotli.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2021 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.codec.compression;
+
+import com.aayushatharva.brotli4j.Brotli4jLoader;
+import io.netty.util.internal.PlatformDependent;
+import io.netty.util.internal.logging.InternalLogger;
+import io.netty.util.internal.logging.InternalLoggerFactory;
+
+public final class Brotli {
+
+  private static final InternalLogger logger = InternalLoggerFactory.getInstance(Brotli.class);
+  private static final ClassNotFoundException CNFE;
+
+  static {
+    ClassNotFoundException cnfe = null;
+
+    try {
+      Class.forName("com.aayushatharva.brotli4j.Brotli4jLoader", false,
```

Comment:
we also use 4 spaces... This needs to be fixed everywhere

Reply:
Will fix.

But frankly, Netty formatting is very complicated to handle, in particular for contributors who can't remember and switch IDE settings for each project they contribute to.
IMHO, Netty should implement something like [spotless](https://github.com/diffplug/spotless) to take care of automatic formatting directly plugged onto maven.